### PR TITLE
Decouple Clinvar submission from users

### LINKED ISSUE #1625: Clinvar submissions
I want all users at Clinical Genetics (cust 002) to see my "variant data" that I want to submit to ClinVar. So we can make a bulk input for submssion of variants (in a "submission CSV file"). 

(tested with another user and she can't see the variant that I marked in the "mirror-clinvar-scout-site" - https://scout.scilifelab.se/cust002/clinvar_submissions)

### PR DESCRIPTION
**Pull request by northwestwitch**
Allow institute-level clinvar submissions. fix #1625

**How to test**:
1. Testable on a local instance of Scout
1. Create a new user with the same institute of test user.
1. Create a clinvar submission containing one or more variants using test user.
1. Login with the other user and make sure the submission is viewable.
1. Add one or more variants to the same submission
1. Remove one or more variants from the same submission.
1. Try to close and reopen the submission.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by dNil
- [x] tests executed by dNil

### CODE REVIEW

**Comment by northwestwitch** [comment]
It's not passing the formatting and pylint tests, but tests and coverage should be fine (could it be that coverage is not triggered because it fails the other 2 tests?). Whatever, ready for review!

**Comment by dnil** [comment]
Nice! Coverage seems to have been triggered alright - perhaps it was just pending due to other jobs - or the push Travis build not being finished yet?

**Comment by northwestwitch** [comment]
> Nice! Coverage seems to have been triggered alright - perhaps it was just pending due to other jobs - or the push Travis build not being finished yet?

Could be, sometimes it takes a while!

File: scout/server/blueprints/cases/views.py [hunk 1]
```diff
@@ -147,7 +147,7 @@ def generate_csv(header, lines):
                 flash('There are no submission objects of type "{}" to include in the csv file!'.format(csv_type),'warning')
 
     data = {
-        'submissions' : controllers.clinvar_submissions(store, current_user.email, institute_id),
+        'submissions' : controllers.clinvar_submissions(store, institute_id),
         'institute_id' : institute_id,
```
[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/blueprints/cases/views.py]: I don't think so. What may happen is that the submission object already in the database has a field named 'user', but it won't be used anywhere

File: tests/utils/test_ensembl_rest_clients.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def test_test_query_biomart_38_xml():
     i = 0
     for i,line in enumerate(client,1):
         ## THEN assert that either the correct gene is fetched or that an HTML page is returned (if the service is down)
-        assert 'ACTR3' in line or line=="<!doctype html>"
+        assert 'ACTR3' in line or line=="<!doctype html>" or "BioMart::Exception::Database" in line
```
[the remark this answers is not in the document]
Comment (northwestwitch) [reply tests/utils/test_ensembl_rest_clients.py]: Thank you! :)

**Comment by northwestwitch** [comment]
This is also ready for merging @moonso , thanks!

**Comment by dnil** [comment]
Ok, no. The tests are ok, and just whitespace weirdness from lint. 
@patrikgrenfeldt - for another PR - lets disable it or if possible flag it as non-mandatory (e.g. run the lint for the output in case there are actual ```-s``` deviations, but always return success).

**Comment by northwestwitch** [comment]
> Good with CHANGELOG.. (testing to re-review to see if that overrides the auto-merge check).

It shouldn't, right?

**Comment by northwestwitch** [comment]
Merging this one!

**Pull request merged**